noir-projects/noir-contracts/contracts/app/nft_contract/src/main.nr

```diff
@@ -197,16 +197,20 @@ pub contract NFT {
         context: &mut PrivateContext,
         storage: Storage<&mut PrivateContext>,
     ) -> PartialNFTNote {
+        let sender = context.msg_sender();
+
         // We create a partial note with unpopulated/zero token id for 'to'
         let partial_note = NFTNote::partial(
             to,
             storage.private_nfts.at(to).storage_slot,
             context,
             to,
-            context.msg_sender(),
+            sender,
         );
 
-        NFT::at(context.this_address())._store_nft_set_partial_note(partial_note).enqueue(context);
+        NFT::at(context.this_address())._store_nft_set_partial_note(sender, partial_note).enqueue(
+            context,
```

Comment:
Cheaper to compute the hash in private and call the public fn with a single param. We're hashing them in priv already anyway to produce the public call kernel request.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/14379/commits/81ba41b0f69ba843c912f28d51cbd0b189a74942